commands/getStats.js

```diff
@@ -0,0 +1,233 @@
+const sqlite = require('sqlite');
+const sqlite3 = require('sqlite3');
+const AsciiTable = require('ascii-table');
+const config = require('../config');
+
+module.exports = {
+  // get number of verified users
+  // linked to STATS command
+  "getNumVerifiedStats" : async function () {
```

Comment:
i think it would be cleaner if we implement the functions outside the `module.exports` body like, 
`const getNumVerifiedStats = async function () {...}` and then at the end of the file, simply pass it in:
`module.exports = {gettNumVerifiedStats, ...}` (with the other functions)